docs/APP-ENV-CONFIG.md

```diff
@@ -85,6 +84,149 @@
 | **AVSCAN_URI`url`                         |                                                          | NA                  | These features are coming soon. Changing/updating may result in unexpected behavior                   |
 | **AVSCAN_APP_ID`url`                      |                                                          | NA                  | These features are coming soon. Changing/updating may result in unexpected behavior                   |
 | **AVSCAN_API_KEY`url`                     |                                                          | NA                  | These features are coming soon. Changing/updating may result in unexpected behavior                   |
+| *FLYWAY_ENABLED`boolean`                   | `true`                                                  | `true/false`          | To enable flyway                                                                                      |
+| FLYWAY_BASELINE_ON_MIGRATE`boolean`       | `true`                                                  | `true/false`          | To enable flyway baseline migration                                                                   |
+| FLYWAY_BASELINEVERSION`int`               | `0`                                                        | `0/1/2...`            | Flyway baseline version                                                                               |
```

Comment:
for all variable types bring to the next line
eg
FLYWAY_BASELINEVERSION`int` 
FLYWAY_BASELINEVERSION<br/>`int`

Reply:
Line break added between variable name and variable type.